main.py

```diff
@@ -1,10 +1,30 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as player_file:
+        player_data = json.load(player_file)
+    for player_name in player_data.keys():
+        info = player_data[player_name]
+        email = info["email"]
+        bio = info["bio"]
+        guild = info["guild"]
+        guild_class = Guild.objects.get_or_create(name=guild["name"], description=guild["description"])
+        race = info["race"]
+        race_class = Race.objects.get_or_create(name=race["name"], description=[race["description"]])
```

Comment:
Similar to the previous comment, unpack the result of `get_or_create` to get the `Race` instance. Use: `race_class, created = Race.objects.get_or_create(...)`. Also, ensure that `description` is passed as a string, not a list.